src/main.less

```diff
@@ -37,6 +37,10 @@
     width: auto;
   }
 
+  tr {
+    height: 23px;
```

Comment:
maybe it should be "min-height: 23px"? Otherwise it will always be 23px, do not know if it is supposed to be able to increase in height in some situations?